From 2543814609a0d9efb607809292e59e12fd05955b Mon Sep 17 00:00:00 2001
From: Henrik Grimler <henrik@grimler.se>
Date: Sun, 29 Sep 2024 13:44:10 +0200
Subject: [PATCH] DONOTMERGE: configs: trats: update values for i9100

i9100 is similar to trats, but needs a different load address. Also
add some convenient commands while we are at it, and
disable (non-working) usb stuff.
---
 configs/trats_defconfig | 17 ++++++-----------
 1 file changed, 6 insertions(+), 11 deletions(-)

diff --git a/configs/trats_defconfig b/configs/trats_defconfig
index 70f7e06ba9c8..486b91137615 100644
--- a/configs/trats_defconfig
+++ b/configs/trats_defconfig
@@ -3,7 +3,7 @@ CONFIG_SKIP_LOWLEVEL_INIT=y
 CONFIG_ARCH_CPU_INIT=y
 CONFIG_SYS_L2CACHE_OFF=y
 CONFIG_ARCH_EXYNOS=y
-CONFIG_TEXT_BASE=0x63300000
+CONFIG_TEXT_BASE=0x40008000
 CONFIG_SYS_MALLOC_LEN=0x5001000
 CONFIG_SYS_MALLOC_F_LEN=0x400
 CONFIG_ARCH_EXYNOS4=y
@@ -15,7 +15,7 @@ CONFIG_ENV_SIZE=0x1000
 CONFIG_ENV_OFFSET=0x7000
 CONFIG_DEFAULT_DEVICE_TREE="exynos4210-trats"
 CONFIG_SYS_MONITOR_LEN=262144
-CONFIG_SYS_LOAD_ADDR=0x44800000
+CONFIG_SYS_LOAD_ADDR=0x4ee00000
 CONFIG_SYS_MEM_TOP_HIDE=0x100000
 CONFIG_FIT=y
 CONFIG_FIT_VERBOSE=y
@@ -29,14 +29,16 @@ CONFIG_SYS_CONSOLE_INFO_QUIET=y
 CONFIG_MISC_INIT_R=y
 CONFIG_SYS_PROMPT="Trats # "
 # CONFIG_CMD_XIMG is not set
-CONFIG_CMD_THOR_DOWNLOAD=y
+CONFIG_CMD_BIND=y
+CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
+CONFIG_CMD_GPIO=y
 CONFIG_CMD_GPT=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
-CONFIG_CMD_USB_MASS_STORAGE=y
 CONFIG_CMD_CACHE=y
 # CONFIG_CMD_SLEEP is not set
+CONFIG_CMD_PMIC=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_OF_CONTROL=y
 CONFIG_ENV_OVERWRITE=y
@@ -54,11 +56,4 @@ CONFIG_MMC_SDHCI_S5P=y
 CONFIG_DM_PMIC=y
 CONFIG_PMIC_MAX8997=y
 CONFIG_USB=y
-CONFIG_USB_GADGET=y
-CONFIG_USB_GADGET_MANUFACTURER="Samsung"
-CONFIG_USB_GADGET_VENDOR_NUM=0x04e8
-CONFIG_USB_GADGET_PRODUCT_NUM=0x6601
-CONFIG_USB_GADGET_DWC2_OTG=y
-CONFIG_USB_GADGET_DWC2_OTG_PHY=y
-CONFIG_USB_GADGET_DOWNLOAD=y
 CONFIG_LIB_HW_RAND=y
-- 
2.50.1

